modules/microshift-cni-multus-troubleshoot.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-cni-multus-troubleshoot_{context}"]
+= Troubleshooting Multus networking
+
+If the settings for multiple networks are not configured properly, pods can fail to start. The following steps can help you solve for a couple common scenarios.
+
+[id="Pod-networking-cannot-be-configured_{context}"]
+== Pod networking cannot be configured
+
+If the Multus CNI plugin cannot apply networking annotations to a pod, the pod does not start. Pods can also fail to start if any of the additional network CNIs fail.
+
+.Example error
+[source,terminal]
+----
+Warning  NoNetworkFound     0s     multus    cannot find a network-attachment-definitio (asdasd) in namespace (default): network-attachment-definitions.k8s.cni.cncf.io "bad-ref-doesnt-exist" not found
+----
+
+In this case, you can take the following steps to trouble CNI failures:
+
+* Verify the values in both the `NetworkAttachmentDefinitions` and the annotations.
+* Remove the annotation to verify whether the pod is created successfully with just the default network. If not, this might indicates a networking problem other than the Multus configuration.
+* If you are a device administrator, you can inspect the `crio.service` or `microshift.service` logs, paying special attention to those that are generated by the `kubelet`.
++
+For example, the following error from the `kubelet` shows that the primary CNI is not running. This situation can be caused either by pods not starting, or because of a CRI-O misconfiguration such as an incorrect `cni_default_network` setting.
```

Comment:
```suggestion
For example, the following error from the `kubelet` shows that the primary CNI is not running. This situation can be caused by pods not starting or because of a CRI-O misconfiguration such as an incorrect `cni_default_network` setting.
```